Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/RdbmsCentralRepo.java

```diff
@@ -1085,8 +1085,19 @@ public CentralRepoAccount getOrCreateAccount(CentralRepoAccountType crAccountTyp
         // Get the account fom the accounts table
         String normalizedAccountID = CentralRepoAccount.normalizeAccountIdentifier(crAccountType, accountUniqueID);
 
-        String insertSQL = "INSERT INTO accounts (account_type_id, account_unique_identifier) "
-                + "VALUES (?, ?) " + getConflictClause();
+        // insert the account.  If there is a conflict, ignore it.
+        String insertSQL;
+        switch (CentralRepoDbManager.getSavedDbChoice().getDbPlatform()) {
+            case POSTGRESQL:
+                insertSQL =  "INSERT INTO accounts (account_type_id, account_unique_identifier) VALUES (?, ?) " + getConflictClause();  //NON-NLS
+                break;
+            case SQLITE:
```

Comment:
@raman-bt This change should not be needed.  You do not need that switch case.  The overload of getConflictCause for each of the subclass should take care of this.  I checked before I made the change.